Recommended node version in readme file (#45)
klaytn/caver-js: README.md

- [npm](https://www.npmjs.com/get-npm)
- [gcc-c++](https://gcc.gnu.org/)

**Note** caver-js can run on Node.js versions 8 and 10, and the recommended versions are:
- lts/carbon ([8.16.0](https://nodejs.org/dist/latest-v8.x/))
- lts/dubnium ([10.16.0](https://nodejs.org/dist/latest-v10.x/))

> If you are already using a different version of the node(for example, node v12), use the Node Version Manager([NVM](https://github.com/nvm-sh/nvm)) to install and use the version of caver-js executable node.

Reviewer: ```suggestion
If you are already using a different version of the node(for example, node v12), use the Node Version Manager([NVM](https://github.com/nvm-sh/nvm)) to install and use the version supported by caver-js.
```

- [npm](https://www.npmjs.com/get-npm)
- [gcc-c++](https://gcc.gnu.org/)

> **Note** The recommended node versions are

Reviewer: ```suggestion
**Note** The recommended node.js versions are
```

그냥 node라고도 많이 쓰나요? web site상에는 node.js라고 쓰여 있는 것 같아서요.

> **Note** The recommended node.js versions are

Author: ```suggestion
**Note** The recommended Node.js versions are
```

- lts/carbon (8.16.0)
- lts/dubnium (10.16.0)

> also, support [AWS lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).

Reviewer: ```suggestion
also, support [AWS lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).
```

여기도 지원한다는 것이 무슨 의미인지 불명확하네요. caver-js를 AWS lambda runtimes에서 쓸수 있다는건데, 어떻게 쓸수 있는지, 대략적인 설명이 있으면 좋을 것 같아요. 람다는 버전 몇을 지원하는데 어떻게 쓰면 된다는 간단한 설명이 있으면 좋을 것 같네요.

Author: Since the Node.js versions supported by Lambda are 8 and 10 (https://docs.aws.amazon.com/en_us/lambda/latest/dg/programming-model.html), the statement that supports lambda May be omitted from this document.
After removing lambda, based on the test results, i described caver-js can be run on node 8 or 10, and recommended version is based on lts.

> If you are using a different node version(for example, node v12), use the node version manager([NVM](https://github.com/nvm-sh/nvm)).

Reviewer: ```suggestion
If you are using a different node.js version(for example, node.js v12), use the node version manager([NVM](https://github.com/nvm-sh/nvm)).
```

NVM을 쓰면 v12를 써도 되는건가요? NVM으로 어떻게 하라는건지가 불명확한데 그 부분 변경 부탁드립니다.

Author: The meaning of this statement is to install and use the version of the node that is supported using NVM. I will add some more content in this section.
